src/etc/check_missing_items.py

```diff
@@ -108,7 +108,7 @@ def check_type(ty):
     elif ty["kind"] == "function_pointer":
         for param in ty["inner"]["generic_params"]:
             check_generic_param(param)
-        check_decl(ty["inner"]["inner"])
+        check_decl(ty["inner"]["decl"])
```

Comment:
Previous implementation looks wrong, other call(s) of check_decl are `["inner"]["decl"]`, and it doesn't look like there should ever be an `["inner"]["inner"]` in the output.